### PR TITLE
Play 2.6 upgrade

### PR DESCRIPTION
**Pull request by davidfurey**
## Why are you doing this?

GDPR

## Changes
* Upgrade to SBT 1.0
* Upgrade to Play 2.6
* Upgrade Cats and Scalaz

### CODE REVIEW

**Comment by johnduffell** [comment]
> presumably the tests are failing because we're waiting for guardian/acquisition-event-producer#26 to be released?

I've just shipped it as v 3.0.0 🚀

File: frontend/app/controllers/CacheBustedAssets.scala [hunk 1]
```diff
@@ -3,8 +3,8 @@ package controllers
 import play.api.mvc.{AnyContent, Action, Controller}
 import views.support.Asset
 
-class CacheBustedAssets() extends Controller{
+class CacheBustedAssets(asset: Assets) extends Controller {
   def at(path: String): Action[AnyContent] =  {
-    controllers.Assets.at("/public", Asset.map(path), true)
+    asset.at("/public", Asset.map(path), true)
```
Comment (johnduffell) [inline frontend/app/controllers/CacheBustedAssets.scala]: missing an `s` (sorry it's the best I can do, not much else to comment on!)

File: frontend/conf/ahc.properties [hunk 2]
```diff
@@ -0,0 +1,91 @@
+org.asynchttpclient.threadPoolName=AsyncHttpClient
```
Comment (johnduffell) [inline frontend/conf/ahc.properties]: what's all this and do we need it in subs too?

File: project/build.properties [hunk 3]
```diff
@@ -1 +1 @@
-sbt.version=0.13.16
+sbt.version=1.0.2
```
Comment (johnduffell) [inline project/build.properties]: might have been possible to do the sbt PR separately, just to keep the size down?  But I realise some things tangle together a lot.

**Comment by johnduffell** [comment]
great, looks good, I wasn't sure what the whole shaded thing was, I don't know if I should add that to subs frontend too?  Other than that only nit picking - go for it! 👍 🚀

File: frontend/conf/ahc.properties (hunk 2 above)
Comment (jacobwinch) [reply frontend/conf/ahc.properties]: I had the same question - there are more details here: https://github.com/guardian/membership-frontend/pull/1780/commits/c8f465673f7dfa512d11b3aa763f180dc3c6ed68

Comment (johnduffell) [reply frontend/conf/ahc.properties]: thanks! not sure if it's worth putting that in a file comment too

**Pull request merged**